docker/container.go

```diff
@@ -327,7 +324,7 @@ func (c *Container) Log(ctx context.Context, l logger.Logger, follow bool) error
 		Follow:     follow,
 		Tail:       "all",
 	}
-	responseBody, err := c.client.ContainerLogs(ctx, c.name, options)
+	responseBody, err := c.client.ContainerLogs(ctx, c.container.Name, options)
```

Comment:
Maybe `c.ID` is better here? From what I understand, we're trying to use IDs rather than names for API calls.